### ISSUE TITLE
Watch2Gether: Display chat message character count

### ISSUE THREAD

**Issue by Serraniel** [Serraniel/AniwatchPlus#91]
Chat messages in w2g chats are limited to 250 characters. It would be nice to have a counter how many characters are left (similar to twitter).

**Comment by Serraniel** [comment]
The issue is blocked by #19, as the `runAfterLocationChange` is required.

**Comment by Serraniel** [comment]
Update: Aniwatch Dev accepted this suggestion, too, so I reduced priority. However, until they only have it on their roadmap and not implemented themselves, this issue will stay open to deliver an implementation by our side.

![image](https://user-images.githubusercontent.com/8461282/97108091-6db9d700-16cb-11eb-9f54-0a0d5d7b8be8.png)

**Comment by Serraniel** [comment]
#19 is closed and development can be continued again.

**Comment by Serraniel** [comment]
Fixed in 0.4.0.
